### PR TITLE
feature: add `PermissionEvaluatorInterface::requireAllPermissions`

### PR DESCRIPTION
**Pull request by dresslerdemos**
### How to review/test

The new method is not used yet and triggering the exceptions may be difficult. Code review should be sufficient.

### Linked PRs (optional)

Must be merged together with https://github.com/demos-europe/demosplan-addon/pull/66.

### PR Checklist

Delete the checkbox if it doesn't apply/isn't necessary.

- [ ] Tests updated/created
- [ ] Update documentation
- [ ] Link all relevant tickets
- [ ] Move the tickets on the board accordingly

### CODE REVIEW

File: demosplan/DemosPlanCoreBundle/Exception/AccessDeniedException.php [hunk 1]
```diff
@@ -37,8 +37,25 @@ public static function missingPermissions(User $user = null, array $permissions
...
 
-    public static function missingPermission(string $permission, User $user = null): self
+        return new self("Der Zugriff auf $permissionName im Addon $addonIdentifier ist nicht gestattet. $additionalUserData");
+    }
+
+    public static function unknownAddonPermission(string $permissionName, string $addonIdentifier, User $user = null): self
+    {
+        $additionalUserData = self::collectAdditionalUserData($user);
+
+        return new self("Es ist keine $permissionName Permission in einem Addons $addonIdentifier bekannt. $additionalUserData");
```
Comment (AlexMenschoid) [inline demosplan/DemosPlanCoreBundle/Exception/AccessDeniedException.php:55]: Typo: Addons -> Addon

**Review by AlexMenschoid — CHANGES_REQUESTED** [review]

File: demosplan/DemosPlanCoreBundle/Exception/AccessDeniedException.php (hunk 1 above)
Comment (dresslerdemos) [inline demosplan/DemosPlanCoreBundle/Exception/AccessDeniedException.php:55]:
```suggestion
        return new self("Es ist keine $permissionName Permission in einem Addon $addonIdentifier bekannt. $additionalUserData");
```

Comment (dresslerdemos) [inline demosplan/DemosPlanCoreBundle/Exception/AccessDeniedException.php:55]:
```suggestion
        return new self("Es ist keine $permissionName Permission in einem Addon $addonIdentifier bekannt. $additionalUserData");
```

**Review by AlexMenschoid — APPROVED** [review]
lgtm, code review only

**Pull request merged**